app/src/main/java/edu/uc/forbesne/shoppingsidekick/ui/main/MainFragment.kt

```diff
@@ -50,25 +50,15 @@ class MainFragment : Fragment() {
         })
     }
 
-        // This will get called when the user clicks on the pop-up window.
-        private fun addCartItem() {
-            // The cart item logic should mostly be in the view model and cart item is not created by the user.
-            // Instead we need only the quantity and a way to identify a product - for all shops - done by UPC
-            // => Commented out until view has these fields
-            /*var quantity = etnQuantity.text.toString()
-            var upc = upc.text.toString() //perhaps this should be stored in a hidden label
-            viewModel.addCartItem(cartItem)*/
-
-
-
-            /*var cartItem = CartItem().apply{
```

Comment:
The item data is not filled and created by the user as in the example of plant in our class.